wave_vision/src/descriptor/brisk_descriptor.cpp

```diff
@@ -0,0 +1,158 @@
+// Libwave Headers
+#include "wave/vision/descriptor/brisk_descriptor.hpp"
+
+namespace wave {
+
+// Default constructor
+BRISKDescriptor::BRISKDescriptor() {
+    // Instantiate cv::BRISK with default values
+    float patternScale = 1.0f;
+    float f = 0.85f * patternScale;
+
+    // radiusList contains the radius (in pixels) of each circle in the sampling
+    // pattern
+    std::vector<float> radiusList = {
+      f * 0.0f, f * 2.9f, f * 4.9f, f * 7.4f, f * 10.8f};
```

Comment:
Why not put these default values into `BRISKDescriptorParams`? Then this constructor would merely have to call the other constructor. I.e. 
```cpp
BRISKDescriptor::BRISKDescriptor() : BRISKDescriptor{BRISKDescriptorParams{}} {}
```